<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mycroft</class>
 <widget class="QMainWindow" name="Mycroft">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mycroft</string>
  </property>
  <widget class="QWidget" name="main_editor">
   <layout class="QVBoxLayout" name="windowlayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_undo"/>
    <addaction name="action_redo"/>
    <addaction name="separator"/>
    <addaction name="action_cut"/>
    <addaction name="action_copy"/>
    <addaction name="action_paste"/>
    <addaction name="action_delete"/>
    <addaction name="separator"/>
    <addaction name="action_selectall"/>
   </widget>
   <widget class="QMenu" name="menu_template">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="actionOpen_2"/>
   </widget>
   <widget class="QMenu" name="menu_tools">
    <property name="title">
     <string>Template</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_find"/>
   </widget>
   <widget class="QMenu" name="menu_window">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_file_open"/>
    <addaction name="separator"/>
    <addaction name="action_file_exit"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_template"/>
   <addaction name="menu_tools"/>
   <addaction name="menu_window"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_file_open">
   <property name="text">
    <string>Open File or Project</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="action_selectall">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="action_find">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionOpen_2">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="action_file_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
